validation/main-ping-validation-with-jsonschema.ipynb

```diff
@@ -2,100 +2,124 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
    "metadata": {
     "collapsed": false
    },
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Unable to parse whitelist (/home/hadoop/anaconda2/lib/python2.7/site-packages/moztelemetry/bucket-whitelist.json). Assuming all histograms are acceptable.\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "from jsonschema import validate, ValidationError\n",
     "import ujson as json\n",
-    "from moztelemetry import get_pings"
+    "from moztelemetry.dataset import Dataset"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Fetch the latest main ping schema."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "import urllib2\n",
+    "f = urllib2.urlopen('https://github.com/mozilla-services/mozilla-pipeline-schemas/raw/master/telemetry/main.schema.json')\n",
+    "schema = json.load(f)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Get some pings."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
    "metadata": {
     "collapsed": true
    },
    "outputs": [],
    "source": [
-    "sample = get_pings(sc, app='Firefox', channel='release',\n",
-    "                   submission_date=\"20160117\", doc_type='main',\n",
-    "                   fraction=0.001)\n",
-    "t10 = sample.take(1000)"
+    "sample = Dataset.from_source(\"telemetry\") \\\n",
+    "    .where(appName='Firefox') \\\n",
+    "    .where(docType='main') \\\n",
+    "    .where(submissionDate=\"20170131\") \\\n",
+    "    .where(appUpdateChannel=\"nightly\") \\\n",
+    "    .where(sourceVersion=\"4\") \\\n",
+    "    .records(sc, sample=0.01)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Do a quick run on a few samples (validate will throw on failure)."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 52,
+   "execution_count": null,
    "metadata": {
     "collapsed": false
    },
    "outputs": [],
    "source": [
-    "import urllib2\n",
-    "f = urllib2.urlopen('https://raw.githubusercontent.com/SamPenrose/mozilla-pipeline-schemas/master/telemetry/main.schema.json')\n",
-    "schema = json.load(f)"
+    "t10 = sample.take(100)"
```

Comment:
Why change from 1000 to 100 sample pings? Can we change the name of the `t10` variable to something more informative?